api/cluster/resource/templater.go

```diff
@@ -79,6 +79,13 @@ const (
 	defaultGRPCPort                  = 9000
 	defaultPredictorPort             = 80
 
+	envPublisherKafkaTopic    = "PUBLISHER_KAFKA_TOPIC"
+	envPublisherKafkaBrokers  = "PUBLISHER_KAFKA_BROKERS"
+	envPublisherEnabled       = "PUBLISHER_ENABLED"
+	envPublisherKafkaLinger   = "PUBLISHER_KAFKA_LINGER_MS"
+	envPublisherKafkaAck      = "PUBLISHER_KAFKA_ACKS"
+	envPublisherSamplingRatio = "PUBLISHER_SAMPLING_RATIO"
+
```

Comment:
Do we also want to expose the [PUBLISHER_KAFKA_CONFIG](https://github.com/caraml-dev/merlin/blob/main/python/pyfunc-server/pyfuncserver/config.py#L31) env var via the Merlin control plane configs?